### ISSUE TITLE
protobuf issue with last mesos master branch

### ISSUE THREAD

**Issue by vieux** [mesos/mesos-go#142]
When I try to start a task with no resource I get the error below.
I guess the protobuf were updated recently, a new required field was added.

```
E0702 13:59:29.810685   20914 messenger.go:367] Failed to unmarshal message &{master@127.0.0.1:10363 mesos.internal.StatusUpdateMessage update:<framework_id:<value:"20150702-205927-16777343-10363-1-0000" > slave_id:<value:"20150702-205927-16777343-10363-1-S0" > status:<task_id:<value:"0c50bab6a78d" > state:TASK_ERROR message:"Task uses no resources" source:SOURCE_MASTER reason:REASON_TASK_INVALID slave_id:<value:"20150702-205927-16777343-10363-1-S0" > timestamp:1.435870769809906e+09 > timestamp:1.435870769809906e+09 > pid:"@0.0.0.0:0"  [10 185 1 10 39 10 37 50 48 49 53 48 55 48 50 45 50 48 53 57 50 55 45 49 54 55 55 55 51 52 51 45 49 48 51 54 51 45 49 45 48 48 48 48 26 37 10 35 50 48 49 53 48 55 48 50 45 50 48 53 57 50 55 45 49 54 55 55 55 51 52 51 45 49 48 51 54 51 45 49 45 83 48 34 94 10 14 10 12 48 99 53 48 98 97 98 54 97 55 56 100 16 7 34 22 84 97 115 107 32 117 115 101 115 32 110 111 32 114 101 115 111 117 114 99 101 115 42 37 10 35 50 48 49 53 48 55 48 50 45 50 48 53 57 50 55 45 49 54 55 55 55 51 52 51 45 49 48 51 54 51 45 49 45 83 48 49 128 213 115 140 105 101 213 65 72 0 80 14 41 128 213 115 140 105 101 213 65 18 10 64 48 46 48 46 48 46 48 58 48]}: proto: required field "Update.{Unknown}" not set
```

**Comment by jdef** [comment]
If you're feeling brave you could try using the v0.22 bindings branch:

https://github.com/mesos/mesos-go/tree/v0_22_bindings

I don't think it's had any testing yet, so you'd be on the bleeding edge.

**Comment by vieux** [comment]
@jdef thanks I saw the PR, I'll test, but I doubt it will fix because it's working with Mesos 0.22.
The change was introduce between 0.22 and 0.23.

**Comment by jdef** [comment]
v0.23 bindings have been merged to master
